react/NotificationInline.js

```diff
@@ -1,3 +1,3 @@
-import NotificationInline from './components/Notification/notificationInline'
+import NotificationInline from './components/Notification/NotificationInline'
```

Comment:
Can we have the code in this file instead?

Reply:
Sure! But then this file would need to be renamed... could that be a breaking change? 🤔